e2e/nomostest/config_sync.go

```diff
@@ -699,8 +683,10 @@ func RootSyncObjectV1Alpha1FromRootRepo(nt *NT, name string) *v1alpha1.RootSync
 	repoURL := nt.GitProvider.SyncURL(repo.RemoteRepoName)
 	sourceFormat := repo.Format
 	rs := RootSyncObjectV1Alpha1(name, repoURL, sourceFormat)
-	if nt.DefaultReconcileTimeout != 0 {
-		rs.Spec.SafeOverride().ReconcileTimeout = toMetav1Duration(nt.DefaultReconcileTimeout)
+	if nt.DefaultReconcileTimeout != nil {
```

Comment:
Should these defaults be moved into the `R*SyncObjectV1Alpha`/`R*SyncObjectV1Beta` to reduce duplication?

Reply:
Those don't have access to NT.
I have considered making them depend on NT to force the defaults into tests that use them directly, but that's a bigger refactor.